packages/components/src/tabs/tablist.tsx

```diff
@@ -26,7 +27,8 @@ export const TabList = forwardRef<
 >( function TabList( { children, ...otherProps }, ref ) {
 	const context = useTabsContext();
 
-	const selectedId = context?.store.useState( 'selectedId' );
+	const tabStore = useStoreState( context?.store );
+	const selectedId = useStoreState( context?.store, 'selectedId' );
```

Comment:
Since we have the whole store state already, we don't need to call the `useStoreState` hook again

```suggestion
	const selectedId = tabStoreState?.selectedId;
```

Reply:
Indeed, done as part of c9b238f3e3faf8f5e6ce711b79c73e10f9841a8f